### DISCUSSION TITLE
Roms folder names

### DISCUSSION THREAD

**Discussion by zebramonkeylizard** [OnionUI/Onion#483] [Polls]
Who thinks the Rom folder names should match the Consoles, e.g., SNES Rom folder should be SNES, NES should be NES, and Sega Genesis should be Genesis(or Gen)? I personally prefer PSX for PlayStation, but that's for another time.

**Comment by schmurtzm** [comment]
Sega Mega Drive you mean ? 😆 
Yes there are better names but these folders are created by MainUI, the main app created by Miyoo so we can't modify it ;)
